inference-engine/src/low_precision_transformations/src/convolution.cpp

```diff
@@ -278,6 +278,11 @@ bool ConvolutionTransformation::transform(TransformationContext &context, ngraph
     ngraph::copy_runtime_info({ convolution, finalDequantization }, finalDequantization);
     updateOutput(context, finalDequantization, convolution);
 
+    // [C, 1, 1] -> [1, C, 1, 1]
+    auto finalDeqConstant = as_type_ptr<opset1::Constant>(NetworkHelper::getConstantInput(finalDequantization));
+    const auto finalConstantWithBatch = NetworkHelper::getConstWithNormalizeShape(finalDequantization, finalDeqConstant);
+    replace_node(finalDeqConstant, finalConstantWithBatch);
+
```

Comment:
Can we simplify and have one method call here, for example:
```cpp
NetworkHelper::normalizeDequantizationShape(finalDequantization);
```
Don't forget about runtime attributes, please. Thanks!

Reply:
Done, Thanks!